scripts/armstrong.py

```diff
@@ -0,0 +1,16 @@
+# armstrong number: 153 = 1^3+5^3+3^3
+
+def is_armstrong(n):
+    s=0
+    a = n
+    while(n>0):
+        p = n %10
+        s = s + (p*p*p)
```

Comment:
Better to write it as `p**3` rather than `p*p*p`

Reply:
Implemented